codechain/run_node.rs

```diff
@@ -238,7 +238,17 @@ pub fn run_node(matches: &ArgMatches) -> Result<(), String> {
             .expect("Current time should be later than unix epoch")
             .subsec_nanos() as usize,
     );
-    clogger::init(&LoggerConfig::new(instance_id)).expect("Logger must be successfully initialized");
+    let email_alarm_config = if config.email_alarm.disable != Some(true) {
```

Comment:
Isn't the type of `disable` Option to support the configuration that was created before this commit, is it?
I think it should be `config.email_alarm.disable == Some(false)`.

Reply:
Since the default instance of the EmailAlarm is created when it does not exist, we don't need to check the Option type of the `disable` field.

I found that other codes read the `disable` field using `Option::unwrap`. I'll change the code to use the `unwrap` for consistency.